MAINTAINERS

```diff
@@ -3,37 +3,37 @@
 # See GOVERNANCE.md for committer versus reviewer roles
 #
 # COMMITTERS
-# GitHub ID, Name, Email address
-"AkihiroSuda","Akihiro Suda","akihiro.suda.cz@hco.ntt.co.jp"
-"crosbymichael","Michael Crosby","crosbymichael@gmail.com"
-"dmcgowan","Derek McGowan","derek@mcgstyle.net"
-"estesp","Phil Estes","estesp@gmail.com"
-"stevvooe","Stephen Day","stevvooe@gmail.com"
-"dchen1107","Dawn Chen","dawnchen@google.com"
-"Random-Liu","Lantao Liu","lantaol@google.com"
-"mikebrow","Mike Brown","brownwm@us.ibm.com"
-"yujuhong","Yu-Ju Hong","yjhong@google.com"
-"fuweid","Fu Wei","fuweid89@gmail.com"
-"mxpv","Maksym Pavlenko","pavlenko.maksym@gmail.com"
-"dims","Davanum Srinivas","davanum@gmail.com"
-"kevpar","Kevin Parsons","kevpar@microsoft.com"
-"kzys","Kazuyoshi Kato","katokazu@amazon.com"
-"samuelkarp","Samuel Karp","me@samuelkarp.com"
+# GitHub ID, Name, Email address,GPG fingerprint
+"AkihiroSuda","Akihiro Suda","akihiro.suda.cz@hco.ntt.co.jp","C020 EA87 6CE4 E06C 7AB9 5AEF 4952 4C6F 9F63 8F1A"
+"crosbymichael","Michael Crosby","crosbymichael@gmail.com",""
+"dmcgowan","Derek McGowan","derek@mcgstyle.net","8C7A 111C 2110 5794 B0E8 A27B F58C 5D0A 4405 ACDB"
+"estesp","Phil Estes","estesp@gmail.com","71AE 6DCD DFF8 C2D1 DDCA EEC9 FE25 9812 6B19 6A38"
+"stevvooe","Stephen Day","stevvooe@gmail.com",""
+"dchen1107","Dawn Chen","dawnchen@google.com",""
+"Random-Liu","Lantao Liu","lantaol@google.com",""
+"mikebrow","Mike Brown","brownwm@us.ibm.com",""
+"yujuhong","Yu-Ju Hong","yjhong@google.com",""
+"fuweid","Fu Wei","fuweid89@gmail.com",""
+"mxpv","Maksym Pavlenko","pavlenko.maksym@gmail.com",""
+"dims","Davanum Srinivas","davanum@gmail.com",""
```

Comment:
mine is `A67E 5FD8 80EB 089F 2317 7967 80D8 3A79 6103 BF59`

Reply:
Added